components/icon/global.css

```diff
@@ -36,6 +36,6 @@
 
   /* See https://github.com/mdn/rari/blob/927fac7cb89c7f7e49dd3ff265c3d74ad89d6490/crates/rari-doc/src/templ/templs/badges.rs#L43-L47 */
   &.icon-nonstandard {
-    mask-image: url("../notecard/icons/warning.svg");
+    mask-image: url("../notecard/icons/triangle-alert.svg");
```

Comment:
@pepelsbey triangle-alert.svg is missing in this branch, explaining the following error:

```
SSR render error: Error: Cannot find module '../icon/triangle-alert.svg'
```